db/seeds.rb

```diff
@@ -16,4 +16,12 @@
   end
 end
 
+puts 'Started seeding Rounds'
+GamePlayer.find_each do |game_player|
+  3.times do
+    round_type = Round.round_types.keys.sample
+    game_player.rounds.create(round_type:)
+  end
```

Comment:
This won't work all the time, as you can sample the same round_type, thus having the unique error.
Better do 
```ruby
Round.round_types.keys.sample(3) do |round_type|
  game_player.rounds.create(round_type:)
end
```